client/webpack.config.js

```diff
@@ -35,11 +40,13 @@ const config = Object.assign(baseConfig, {
   context: resolve(__dirname),
 
   entry: {
-    'webpack-bundle': [
+    'moments-app_bundle': glob.sync('./app/bundles/momentsApp/startup/*'),
```

Comment:
will we be separating bundles for momentsApp / alliesApp and etc going forward? Or only moments app due to the chart.js requirements?

Reply:
We should let webpack dynamically determine chunking. If we use UI routing, then this becomes far more feasible. Right now we're manually creating chunk entries, but that's not scalable.